website/source/docs/builders/openstack.html.md

```diff
@@ -157,6 +157,41 @@ builder.
 -   `security_groups` (array of strings) - A list of security groups by name to
     add to this instance.
 
+-   `source_image_filter` (object) - Filters used to populate filter options.
+    Example:
+
+    ``` json
+    {
+        "source_image_filter": {
+            "filters": {
+                "name": "ubuntu-16.04*",
+                "visibility": "protected",
+                "owner": "d1a588cf4b0743344508dc145649372d1",
+                "tag": "prod"
```

Comment:
`tag` should be named `tags` and take a list of tags.

Reply:
`tag` is currently implementing by taking a comma delimited list within a string, e.g. "prod,ready". 

Should it instead take `["prod", "ready"]`? I'll get that updated once you confirm.

Reply:
Issue: ListOpts.Tags using the `q:"tag"` field. I can refactor the code to ignore this, but that is how I chose the keys for the configuration. It is based off the OpenStack ImageService docs. I can however update the field to take a slice. This will be the only non-string type in the map and I will need to refactor it to take in `map[string]interface{}` instead. 

I'll go ahead and work on the string->slice change, but I suggest not changing the `tag` name since the scheme is meant to duplicate the OpenStack documentation and reduce fragmentation.

Reply:
It should be named `tags` and take a list since `"tag": ["prod", "ready" ]` translates into multiple `tag` parameters: `?tag=prod&tag=ready`.

Reply:
That is the only query parameter that can be used multiple times.

Reply:
Fixed.